src/main/java/org/springframework/data/couchbase/core/ReactiveRemoveByQueryOperationSupport.java

```diff
@@ -100,11 +101,10 @@ public Flux<RemoveResult> all() {
 				} else {
 					TransactionQueryOptions opts = OptionsBuilder
 							.buildTransactionQueryOptions(buildQueryOptions(pArgs.getOptions()));
-					ObjectNode convertedOptions = com.couchbase.client.java.transactions.internal.OptionsUtil
-							.createTransactionOptions(pArgs.getScope() == null ? null : rs, statement, opts);
+					CoreQueryContext queryContext = CollectionIdentifier.DEFAULT_SCOPE.equals(rs.name()) ? null : CoreQueryContext.of(rs.bucketName(), rs.name());
 					return transactionContext.get().getCore()
-							.queryBlocking(statement, template.getBucketName(), pArgs.getScope(), convertedOptions, false)
```

Comment:
I forgot that Spring was using these internals directly.  Let's get these methods annotated with the new annotation so breakage like this doesn't happen again.